src/deep_clone.ts

```diff
@@ -24,8 +22,7 @@ function clone_serializable(obj)
     return objectInputStream.readObject();
 }
 
-function serialize_parcelable(obj)
-{
+function serialize_parcelable(obj: Java.Wrapper) {
```

Comment:
don't we know the ret-type of this func? (convertible to Uint8Array? convert it and return a known typescrict value)